pkg/apiserver/apiserver.go

```diff
@@ -15,6 +15,7 @@
 package apiserver
 
 import (
+	"github.com/vmware-tanzu/antrea/pkg/controller/apiserver/handlers/endpoint"
```

Comment:
Please move this line after the blank line in import(), together with other antrea related packages.